client/src/views/country_info.js

```diff
@@ -3,7 +3,16 @@ const CountryInfo = function () {
 }
 
 CountryInfo.prototype.render = function (country) {
+	console.log(country);
```

Comment:
remove this please

Reply:
This line of code is integral to our codebase.

Reply:
better remove it or ill drop your database

Reply:
It's too late. It's been committed. Enjoy having the country you selected logged to console every time you select it in the dropdown!